experimental/grpc_attestation/src/main.rs

```diff
@@ -82,10 +81,10 @@ pub struct Opt {
 // Example message sent to the client.
 const EXAMPLE_MESSAGE: &str = "Example message";
 
-fn request_handler(request: &[u8]) -> Vec<u8> {
-    let parsed_request = String::from_utf8(request.to_vec()).expect("Couldn't parse request");
+async fn request_handler(request: Vec<u8>) -> anyhow::Result<Vec<u8>> {
```

Comment:
Is there a reason for switching from `&[u8]` to `Vec<u8>`?

Reply:
This is because I changed the type of `handler` in `RequestHandler`. The signature of the closure now requires `Vec<u8>`. I could not make it work with `&[u8]`. There is either an issue with lifetimes, or unsatisfying the `Clone` bound.